packages/starlight/utils/user-config.ts

```diff
@@ -255,6 +255,23 @@ const UserConfigSchema = z.object({
 				.describe(
 					'Define whether headings in content should be rendered with clickable anchor links. Default: `true`.'
 				),
+			/**
+			 * Define additional directories where files should be processed by Starlight’s Markdown pipeline which includes support for:
+			 *
+			 *  - Clickable heading anchor links
+			 *  - Asides Markdown directive syntax
+			 *  - RTL support for code blocks
+			 *
+			 * Supports local directories relative to the root of your project, e.g. './src/data/comments/'.
+			 * Content of the `docs` content collection is always processed by Starlight’s Markdown pipeline.
```

Comment:
This is very helpful, but I’m a bit nervous about listing the supported features here, because I suspect we’d quickly forget to update it when adding more Markdown features. Maybe we can omit it?
```suggestion
			 * Define additional directories where files should be processed by Starlight’s Markdown pipeline.
			 *
			 * Supports local directories relative to the root of your project, e.g. './src/data/comments/'.
			 * Content of the `docs` content collection is always processed by Starlight’s Markdown pipeline.
```